### PR TITLE
Set minimum for CCTP withdraws

### PR DESCRIPTION
**Pull request by rosepuppy**

### CODE REVIEW

**Review by aforaleka — APPROVED** [review]

File: src/views/forms/AccountManagementForms/WithdrawForm.tsx [hunk 1]
```diff
@@ -329,6 +334,12 @@ export const WithdrawForm = () => {
           },
         });
       }
+      if (
+        !debouncedAmountBN.isZero() &&
+        MustBigNumber(debouncedAmountBN).lte(MIN_CCTP_TRANSFER_AMOUNT)
+      ) {
+        return 'Amount must be greater than 10 USDC';
```
Comment (aforaleka) [inline src/views/forms/AccountManagementForms/WithdrawForm.tsx:341]: nit would be localization and using MIN_CCTP_TRANSFER_AMOUNT! but all good as followup

File: src/constants/numbers.ts [hunk 2]
```diff
@@ -22,4 +22,5 @@ export enum NumberSign {
 
 // Deposit/Withdraw
 export const MAX_CCTP_TRANSFER_AMOUNT = 1_000_000;
+export const MIN_CCTP_TRANSFER_AMOUNT = 10;
```
Comment (jaredvu) [inline src/constants/numbers.ts:25]: Would these two variables be better in shared configs so mobile can leverage?

Comment (jaredvu) [reply src/constants/numbers.ts:25]: max price impact may also be helpful

**Review by jaredvu — APPROVED** [review]

File: src/constants/numbers.ts (hunk 2 above)
Comment (rosepuppy) [reply src/constants/numbers.ts:25]: yes will follow up to have the validation be in abacus!

**Pull request merged**